### ISSUE TITLE
api.MessagePort - Node js doesn't appeared in it

### ISSUE THREAD

**Issue by ThumusLive** [mdn/browser-compat-data#9473]
#### What information was incorrect, unhelpful, or incomplete?
The table with version on it, since v15.0.0, nodejs added MessagePort and I didn't see it in the table
Link : https://nodejs.org/dist/latest-v15.x/docs/api/worker_threads.html#worker_threads_class_messageport
Images:
![image](https://user-images.githubusercontent.com/42680097/111217284-b704b780-85d5-11eb-9186-932a77bdeac3.png)
![image](https://user-images.githubusercontent.com/42680097/111217482-e3b8cf00-85d5-11eb-8307-ea1d54cb7c55.png)

#### What did you expect to see?
NodeJS appeared in the table
#### Did you test this? If so, how?
Uh I don't need it

<details>
<summary>MDN page report details</summary>

* Query: `api.MessagePort`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/MessagePort
* Report started: 2021-03-15T20:23:35.096Z

</details>

**Comment by ddbeck** [comment]
Thanks for reporting this, @ThumusLive.

We do have this data in our a data set (for example, in [`MessagePort.json`](https://github.com/ddbeck/browser-compat-data/blob/main/api/MessagePort.json)), but, unfortunately, this data is not shown on MDN.

It's a known, but undocumented, issue that Node.js data is never shown on MDN Web API pages, only JavaScript pages. I've opened an issue, https://github.com/mdn/yari/issues/3410, to at least track that particular problem. I'm not sure when that might change, but I've made sure it's filed in the right place to see to it being rendered in the future.

Since there's nothing more to be done for the data, I'm going to close the issue here. Thank you again!
